frontend/packages/text-editor/src/RightMenu.tsx

```diff
@@ -52,6 +54,10 @@ export const RightMenu = ({
     setLangCodeToDelete((prevState) => (prevState === langCode ? null : langCode));
   };
 
+  useEffect(() => {
+    localStorage.setItem('selectedLanguages', JSON.stringify(selectedLanguages));
+  }, [selectedLanguages]);
```

Comment:
This useEffect can probably be removed since you already have one in the TextEditor component
```suggestion
```

Reply:
removed